shm/coordinates.cpp

```diff
@@ -0,0 +1,26 @@
+#include "coordinates.hpp"
+
+#include <cmath>
+
+int constexpr DEFAULT_POSITION_X = 0;
+int constexpr DEFAULT_POSITION_Y = 0;
```

Comment:
One times you declare const values with lower cases and other time with upper cases. Please use one style.